keyboards/kprepublic/bm40hsrgb/keymaps/simchee/keymap.c

```diff
@@ -0,0 +1,56 @@
+/* Copyright 2022 Sinan Okman (@simchee)
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+#include QMK_KEYBOARD_H
+
+enum layers {QWERTZ, LOWER, RAISE, FN1};
```

Comment:
For readability: 
```suggestion
enum layers {
    QWERTZ, 
    LOWER, 
    RAISE, 
    FN1,
};
```

Fun fact, if you add a comma after the last entry, it prevents the formatter from compressing it to a single line.